utils/symbolicate-linux-fatal

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# symbolicate-linux-fatal - Symbolicate Linux stack traces -*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+#
+# Symbolicates fatalError stack traces on Linux. Takes the main binary
+# and a log file containing a stack trace. Non-stacktrace lines are output
+# unmodified. Stack trace elements are analysed using reconstructed debug
+# target matching the original process in where shared libs where mapped.
+#
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import argparse
+import lldb
+import subprocess
```

Comment:
```
./utils/symbolicate-linux-fatal:28:1: I201 Missing newline before sections or imports.
./utils/symbolicate-linux-fatal:29:1: I100 Import statements are in the wrong order. import subprocess should be before import lldb
./utils/symbolicate-linux-fatal:29:1: I201 Missing newline before sections or imports.
```

`import` block should be:
```python
from __future__ import print_function

import argparse
import subprocess

import lldb
```
because `lldb` is not a standard library.